keadm/cmd/keadm/app/cmd/util/cloudinstaller.go

```diff
@@ -63,6 +64,10 @@ func (cu *KubeCloudInstTool) InstallTools() error {
 			cloudCoreConfig.Modules.CloudHub.AdvertiseAddress = strings.Split(cu.AdvertiseAddress, ",")
 		}
 
+		if cu.DNSName != "" {
+			cloudCoreConfig.Modules.CloudHub.DNSNames = strings.Split(cu.DNSName, ",")
```

Comment:
We need validations somewhere on the user input

Reply:
@kevin-wangzefeng  sure, I will be create a new pr to do it.